tests/functions/test_process_text.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+import pandas as pd
+import pytest
+from pandas.testing import assert_frame_equal
+
+from janitor.functions import process_text
+
+
+def test_str_split():
+    "Test Pandas string split method"
+
+    df = pd.DataFrame(
+        {"text": ["a_b_c", "c_d_e", np.nan, "f_g_h"], "numbers": range(1, 5)}
+    )
+
+    result = process_text(
+        df=df, column="text", string_function="split", pat="_"
+    )
```

Comment:
@samukweku I think we use the chaining form in the other tests. Unless we haven't been consistent about it? 

```suggestion
    result = df.process_text(
        column="text", string_function="split", pat="_"
    )
```